tests/lib/c_lib/src/main.c

```diff
@@ -160,6 +157,20 @@ ZTEST(test_c_lib, test_stdint)
 #endif
 }
 
+/**
+ *
+ * @brief Test time_t to make sure it is at least 64 bits
+ *
+ */
+ZTEST(test_c_lib, test_time_t)
+{
+#ifdef CONFIG_EXTERNAL_LIBC
```

Comment:
It feels it would be better to have this filters based on the new NATIVE_LIBC. (meaning, have this PR on top of that commit)

Reply:
yup, but that's not been merged yet, and otherwise these two series are fairly separate. If one of these is merged before the other, I'll make sure to fix the pending one up.

Reply:
Ok. Note you can have both PRs include that shared commit. Git will realize the commit is common, and properly drop it when merging the 2nd PR.